site-packages/integralstor_utils/config.py

```diff
@@ -115,6 +117,18 @@ def get_minimum_user_uid_gid(type='user'):
         return id, None
 
 
+def get_normal_users():
+    try:
+        ret, err = command.get_command_output("awk -F'[/:]' '{if ($3 >= 1000 && $3 < 65534) print $1}' /etc/passwd")
```

Comment:
But all the install time users should be treated as integralstor system users anyway. I'm assuming the get_normal_users is only asking for integralview created users